README.rst

```diff
@@ -20,7 +20,6 @@ This library should work with any standards compliant OpenID Connect provider.
 
 It has been tested with:
 
-* `Google+ Login <https://developers.google.com/accounts/docs/OAuth2Login>`_
```

Comment:
I think we should try to keep existing features if we can. Flask-OIDC is used outside Fedora.

Reply:
We've not tested latest version with that, so best not leave it in. I think google+ is no longer even a service? I'll try put an example app together which makes use of a service like google+ and then readd at a later date though!